.gitignore

```diff
@@ -11,9 +11,9 @@
 /tmp/
 
 # ignore local ide settings
-/.vagrant/
-/.idea/
-/.github/
+.vagrant/
+.idea/
+.github/
```

Comment:
This folder is essential because it contains the CI settings which should be tracked by git! I removed the line already in [this pr](https://github.com/hpi-swt2/connections-portal/pull/110)